src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -30,10 +30,7 @@ import { Client } from 'node-appwrite';
 export default async ({ req, res, log, error }) => {
   // Why not try the Appwrite SDK?
   //
-  // const client = new Client()
-  //    .setEndpoint('https://cloud.appwrite.io/v1')
-  //    .setProject(process.env.APPWRITE_FUNCTION_PROJECT_ID)
-  //    .setKey(process.env.APPWRITE_API_KEY);
+  // const client = new Client().setKey(req.headers['x-appwrite-key']); Set API key; it's automatically provided by Appwrite
```

Comment:
Let's keep the multiline like before so we can make it clear the comment is for the `setKey()` part.